src/index.php

```diff
@@ -1,4 +1,13 @@
 <?php
-
+namespace TheRealMVP;
 require_once '../vendor/autoload.php';
 
+
```

Comment:
I don't think this should just be in index.php. Think the query should be inside the getData function?

Reply:
Think this is OK... Ultimately you'll need to link into index.